gitlabform/processors/abstract_processor.py

```diff
@@ -20,6 +22,7 @@ def __init__(self, configuration_name: str, gitlab: GitLab):
                 [str, list[dict[str, Union[str, int]]], list[dict[str, int]]], bool
             ],
         ] = {}
+        self.gl: Gitlab = GitlabWrapper(self.gitlab)._gitlab
```

Comment:
@gdubicki - I added this here so that other features can easily use the `python-gitlab` library using the `Gitlab` object created here. This should be easily accessible so that others can help migrate rest of the features to `python-gitlab` library. That was my goal. Is there a better way to do this?

Also, should I rename `self.gl` to `self.Gitlab`? I'm not sure if that would be confusing with `self.gitlab`, which is gitlabform's internal implementation of connecting with GitLab API endpoint. Please correct me if I'm mistaken.

Reply:
Hey, I think it's a great idea. The name is fine, we can refactor it in the future when we remove the internal library completely.

Reply:
Sounds good. Thanks for reviewing.